lib/generators/decidim/install_generator.rb

```diff
@@ -121,10 +121,11 @@ def recreate_db
         rails "db:environment:set", "db:drop" unless ENV["CI"]
         rails "db:create"
 
+        rails "db:migrate"
+
         if options[:seed_db]
-          rails "db:migrate", "db:seed"
-        else
-          rails "db:migrate"
+          say "Seeding the database, this can take a while..."
+          rails "db:seed"
```

Comment:
The previous idea was to run migrations and seeds in the same process, since that exposes bad usage of models inside migrations, because the not fully migrated versions of the real models are used in seeds.

Reply:
I should have added a comment in the code to explain why I changed this.

Reply:
You mean `rails "db:migrate", "db:seed"`? I can change this back, I wanted to add some feedback about seeding because I first thought the process was stuck somehow, but with your PR this can be solved I guess.

Reply:
Yes, `rails "db:migrate", "db:seed"` runs everything in the same process, so that if we are using models in migrations the bad models will stay loaded in the `seed` task, and we can detect them more easily.

Maybe we can add logging to the `rails` method via a parameter or something? I like clean test output, but I understand your feeling, it happened to me too. Hopefully #1924 alleviates this, indeed.

Reply:
I don't think we can, since we'd need to find a rails command that only outputs text and I don't think it exists. It's good, I'll leave it as is.

Reply:
I mean, as it is on master.